launchable/commands/subset.py

```diff
@@ -426,6 +428,12 @@ def stdin(self) -> Union[TextIO, List]:
             they didn't feed anything from stdin
             """
             if sys.stdin.isatty():
+                """
+                Note: If pts v2 (state if HANDS_ON_LAB_V2), we allow empty stdin
+                """
```

Comment:
There must be some misunderstandings about Python docstring (the triple double quotes). AFAIK, there's no valid use case in which that appears at the top of the if statement, plus, line 420-422 above was added before the original docstring of the `stdin` method.